src/ApplicationDefaultCredentials.php

```diff
@@ -165,7 +167,88 @@ public static function getCredentials(
         }
 
         if (is_null($creds)) {
-            throw new \DomainException(self::notFound());
+            throw new DomainException(self::notFound());
+        }
+        if (!is_null($cache)) {
+            $creds = new FetchAuthTokenCache($creds, $cacheConfig, $cache);
+        }
+        return $creds;
+    }
+
+    /**
+     * Obtains an AuthTokenMiddleware that uses the default FetchAuthTokenInterface
```

Comment:
This sentence doesn't make total sense. :)

edit: I see now it's copied from `getMiddleware` and `getCredentials`. Can we fix all instances?

Can this also highlight how the result differs from `getMiddleware`?